pkg/kubectl/cmd/testdata/edit/testcase-not-update-annotation/2.request

```diff
@@ -1,7 +1,14 @@
 {
-  "metadata": {
-    "labels": {
-      "new-label": "new-value"
-    }
-  }
+	"metadata": {
+		"labels": {
+			"new-label": "new-value"
+		}
+	},
+	"spec": {
+		"$setElementOrder/ports": [
```

Comment:
at the very least, no order and no content change. if you touch one of the items in the list, I could see it being useful to send the client's view of the order (in case you changed a field within the object that depends on the current order as seen by the client)